### ISSUE TITLE
Add an option to make viper spit out debug output

### ISSUE THREAD

**Issue by zneix** [spf13/viper#1152]
**Is your feature request related to a problem? Please describe.**
I am in the process of implementing viper as configuration manager for my Golang project. While debugging, I've found it very hard to notice why does viper not want to read my config files and it turned out I specified wrong config path in my code - I wouldn't figure this out if I didn't change jww's output level in viper's source code. I find it highly problematic in some cases to not have any option to get debug output for viper without having to modify its source.

**Describe the solution you'd like to see**
A method like `viper.SetLogLevel()` / `viper.BeVerbose()` or something along these lines, could be useful while working with viper as explained in my case above.

**Describe alternatives you've considered**
Modifying viper's source code and using `git reset --hard` every time you're done debugging.

**Additional context**
N/A

**Comment by vietvudanh** [comment]
I was frustrating with this too. As I can see, viper is using `jww` for logging, but I cannot see how to enable it yet.

**Comment by tatablack** [comment]
I agree, it would be nice to have some global setting (e.g. an environment variable) to define Viper's default logging level.

As a workaround (as @vietvudanh noticed), it's possible to set `jww`'s logging level from your own code, without touching Viper's source (check `jww`'s [own README](https://github.com/spf13/jwalterweatherman#changing-the-thresholds) for more details:

```go
import (
	jww "github.com/spf13/jwalterweatherman"
)

jww.SetLogThreshold(jww.LevelTrace)
jww.SetStdoutThreshold(jww.LevelTrace)
```

**Comment by vietvudanh** [comment]
> I agree, it would be nice to have some global setting (e.g. an environment variable) to define Viper's default logging level.
> 
> As a workaround (as @vietvudanh noticed), it's possible to set `jww`'s logging level from your own code, without touching Viper's source (check `jww`'s [own README](https://github.com/spf13/jwalterweatherman#changing-the-thresholds) for more details:
> 
> ```go
> import (
> 	jww "github.com/spf13/jwalterweatherman"
> )
> 
> jww.SetLogThreshold(jww.LevelTrace)
> jww.SetStdoutThreshold(jww.LevelTrace)
> ```

I ended up with setting log level in my config file, then using code to set `jww` threshold.
